Clicker/SectionControllers/PollSectionController.swift

```diff
@@ -9,15 +9,24 @@
 import Foundation
 import IGListKit
 
-protocol PollSectionControllerDelegate {
-    var userRole: UserRole { get }
-}
 
 class PollSectionController: ListSectionController {
     
+    // MARK: these refrences must be passed to each cell in the section
+    var session: Session!
+    var userRole: UserRole!
+    var socket: Socket!
+    var askedCardDelegate: AskedCardDelegate!
+    
+    
     var poll: Poll!
     let widthScaleFactor: CGFloat = 0.9
     
+    override init() {
+        super.init()
+        self.inset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 10)
```

Comment:
what is this `inset` being used for?

Reply:
The inset between the cards. Without it the cards appeared directly in contact with each other.